include/aos/crypto/crypto.hpp

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
This implementation should be in `aos/common/crypto/mbedtls/crypto.hpp`

Reply:
Done